ReSharper.FSharp/src/FSharp.Psi.Features/src/Parsing/FSharpImplTreeBuilder.fs

```diff
@@ -149,7 +148,7 @@ type FSharpImplTreeBuilder(lexer, document, decls, lifetime, projectedOffset, li
                 match kind with
                 | TyconDelegate(synType, _) ->
                     match members with
-                    | SynMemberDefn.ImplicitCtor _ as ctor :: _ -> x.ProcessTypeMember(ctor)
+                    | SynMemberDefn.ImplicitCtor _ :: _ -> x.ProcessTypeMembers(members |> List.take 1)
```

Comment:
It'll end up with two member lists in many types, and the grammar rule doesn't support it.